lib/oli/grading.ex

```diff
@@ -94,7 +94,7 @@ defmodule Oli.Grading do
       timestamp: timestamp,
       scoreGiven: resource_access.score,
       scoreMaximum: resource_access.out_of,
-      comment: "",
+      comment: "Score default comment",
```

Comment:
Not sure if this is fine or if we should set something different here, or think in take another approach